tests/validation/tests/v3_api/resource/cluster-ha.yml

```diff
@@ -28,25 +28,43 @@ services:
                enforce-version: latest
              exemptions:
                namespaces: 
-               - ingress-nginx
-               - kube-system
-               - cattle-system
-               - cattle-epinio-system
-               - cattle-fleet-system
-               - longhorn-system
-               - cattle-neuvector-system
-               - cattle-monitoring-system
-               - rancher-alerting-drivers
-               - cis-operator-system
+               - cattle-provisioning-capi-system
+               - calico-apiserver
+               - calico-system
+               - cattle-alerting
                - cattle-csp-adapter-system
+               - cattle-elemental-system
+               - cattle-epinio-system
                - cattle-externalip-system
+               - cattle-fleet-local-system
+               - cattle-fleet-system
                - cattle-gatekeeper-system
-               - istio-system
+               - cattle-global-data
+               - cattle-global-nt
+               - cattle-impersonation-system
+               - cattle-istio
                - cattle-istio-system
+               - cattle-logging
                - cattle-logging-system
-               - cattle-windows-gmsa-system
+               - cattle-monitoring-system
+               - cattle-neuvector-system
+               - cattle-prometheus
+               - cattle-resources-system
```

Comment:
we don’t need `cattle-provisioning-capi-system`, `cattle-resources-system` to be added in the exemption list as per https://ranchermanager.docs.rancher.com/v2.7/pages-for-subheaders/rke1-hardening-guide

is there a reason why we are adding those namespaces?

Reply:
these were copied directly from the example here: https://ranchermanager.docs.rancher.com/reference-guides/rancher-security/psa-restricted-exemptions

Reply:
capi was added after debugging an additional problem found as part of this ticket, and will be added to the docs